content/cloud-computing/concepts/cloud-services/cloud-services.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: 'Cloud Services'
+Description: 'There are many different cloud-computing services, but the big three are Microsoft Azure, Amazon Web Services (AWS) and Google Cloud Platform (GCP).'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Cloud Computing'
+  - 'Networking'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
```

Comment:
Overall comments: I think this is pretty good, although there are certain areas, mainly the paragraphs introducing each platform that are quite dense. Would recommend breaking some of that up and reducing the technical complexity.